eventorder_test.go

```diff
@@ -0,0 +1,403 @@
+//go:build integration
```

Comment:
```suggestion
```

I would prefer the usage of `SLOW` environment variable for those tests. build tags have some practical issues. A lot of go tools and IDEs / code editors ignore those files if not being explicit setup. The problem is that they are skipped silently without any indication. 

This is why we are using the `SLOW` environment variable; let's be consistent on this - or discuss it again in the next go-chapter meeting.

Reply:
I am not fond of build tags either, but we need to change stuff in various places for the proposed changes to happen.
FYI, we have a lot of other integration tests, apart from docker_test which might be classified as SLOW :)

Reply:
You are right, I would suggest we start with [removing](https://github.com/rudderlabs/rudder-server/pull/2346) build tags and then we can identify and mark slow ones